lib/Onyx.js

```diff
@@ -493,11 +493,13 @@ function remove(key) {
  *
  * @private
  * @param {Error} error
- * @param {Function} ionMethod
+ * @param {Function} onyxMethod
  * @param  {...any} args
  * @return {Promise}
  */
-function evictStorageAndRetry(error, ionMethod, ...args) {
+function evictStorageAndRetry(error, onyxMethod, ...args) {
+    logInfo(`Handled error: ${error}`);
```

Comment:
I think 'Handled error' could be made a bit more specific, right? Can we use `handled Onyx error`, or `Clearing Onyx storage after error: ...`?

Reply:
we tag them with `[Onyx]`

https://github.com/Expensify/react-native-onyx/blob/224e2ba03f2365726b8c1d0785e750c54a85332b/lib/Logger.js#L28

Reply:
Ah fantastic, thanks